inventory.yml.sample

```diff
@@ -301,3 +304,21 @@ all:
 
               # # Uncomment to set an alternate installation disk device for the node
               # installation_disk_path: /dev/sdb
+
+        day2_workers:
+          vars:
+            role: worker
+            vendor: HPE # This example uses baremetal worker nodes
+          hosts:
+            worker1:
```

Comment:
`worker1` is already defined under `workers` in this file, which will lead to a silent name conflict. 

Unfortunately, the names have to be unique, as Ansible is flattening the hosts from all groups into a single list of hosts (`workers.worker1` and `day2_workers.worker1` become one `worker1` host).

Assume the nodes' vars are set to the following (values taken from the proposed sample inventory):

```
workers.worker1:
  mac: 3C:FD:FE:78:AB:03
  vendor: Dell
  # ...
    
day2_workers.worker1:
  mac: 3C:FD:FE:78:AB:05
  vendor: HPE
  # ...
```

The `worker1` hostvars will be:
```shell
$ ansible-inventory --host worker1 -i proposed-sample-inventory.yml | grep '"vendor"\|"mac"'
"mac": "3C:FD:FE:78:AB:05",
"vendor": "Dell",
```

As you can see, `worker1` is in disarray. The mac address is that of the `day2_workers.worker1` node, but the value of `vendor` was taken from `workers.worker1`.

Reply:
Thanks for the review, have updated.